### DISCUSSION TITLE
problemi con installazione debian su: Intel Alder Lake-N100

### DISCUSSION THREAD

**Discussion by litrico85** [indomus/forum#3019] [HUB: Home Assistant]
Ciao a tutti, mi presento in quanto è il mio primo post, mi chiamo Carlo e vorrei iniziare con il mondo di HA e già nemmeno ho iniziato ed ho il primo intoppo :) 
sto provando ad installare debian seguendo la guida presente qui su indomus ma purtroppo quando arrivo allo step della connessione non trova nessuna connessione, ne con wifi in un primo momento, ne con la lan in secondo momento (per fare questa prova ho creato una postazione a terra vicino il router tra le grida di mia moglie). Cosa mi consigliate di fare? procedo con l'installazione e poi provo a risolvere, oppure è meglio fare qualcosa prima? PS: sul windows pre-installato la rete la trova, quindi la scheda di rete funziona. Grazie a tutti in anticipo per la risposta.

**Comment by trebestie** [comment]
Il wifi può essere ostico da configurare, la scheda di rete, salvo improbabili assenze di driver, dovrebbe funzionare senza problemi.
Magari per qualche motivo il tuo router non fornisce il dhcp e quindi devi inserire manualmente l'indirizzo ip oppure sei già su ipv6.
Non sono pratico di Debian e systemd, però leggendo [qui](https://wiki.debian.org/NetworkConfiguration) sicuramente risolvi.
Se posti qualche info aggiuntiva tipo log/dmesg/journalctl magari si capisce dove sta il problema

**Reply by litrico85** [reply]
Grazie mille, alla fine siccome domani cambio casa aspetto di vedere se funziona con il nuovo router e poi ti faccio sapere
